source/material_model/visco_plastic.cc

```diff
@@ -670,6 +749,21 @@ namespace aspect
                                    "surface: $v_{y}=\\sigma_{y}/(2{\\varepsilon}_{ii})$. "
                                    "This form of plasticity is commonly used in geodynamic models "
                                    "See, for example, Thieulot, C. (2011), PEPI 188, pp. 47-68. "
+                                   "\n"
+                                   "The user now has the option to linearly reduce the cohesion and "
```

Comment:
I would leave out the *now*. This description is for the current state, it does not need to contain history information. The history information can be packed in doc/modules/changes.h, which keeps the documentation itself more concise.